dependency/controller.py

```diff
@@ -34,9 +34,14 @@ def create_dependency(workspace_id: str):
     }), 200
 
 @blueprint.get('/')
-def get_all_dependecies(workspace_id: str):
+def get_dependecies(workspace_id: str):
```

Comment:
oh my goodness, how do I keep doing this?
```suggestion
def get_dependencies(workspace_id: str):
```